lib/plugins/aws/lib/naming.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
```

Comment:
I would put the `naming.js` file in a `utils` dir (`lib/plugins/aws/utils/naming.js`) as it's more a utility functionality.